src/mlpack/bindings/cli/in_place_copy.hpp

```diff
@@ -0,0 +1,88 @@
+/**
+ * @file in_place_copy.hpp
+ * @author Ryan Curtin
+ *
+ * Use template metaprogramming to set filenames correctly for in-place copy
+ * arguments.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_BINDINGS_CLI_IN_PLACE_COPY_HPP
+#define MLPACK_BINDINGS_CLI_IN_PLACE_COPY_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace bindings {
+namespace cli {
+
+/**
+ * This overload is called when nothing special needs to happen to make
+ * something an in-place copy.
+ *
+ * @param d ParamData object to get parameter value from.
```

Comment:
Missing documentation for the `input` parameter.

Reply:
Ah, nice catch; added in fe3279904.